module/os/linux/spl/spl-generic.c

```diff
@@ -46,6 +46,7 @@
 #include <sys/strings.h>
 #include <linux/kmod.h>
 #include "zfs_gitrev.h"
+#include <sys/zfs_context.h>
```

Comment:
Since this is Linux specified kernel code let's `#include <linux/mod_compat.h>` directly here.